Orange/widgets/tests/base.py

```diff
@@ -215,31 +221,42 @@ def send_signal(self, input, value, *args, widget=None, wait=-1):
             else:
                 raise ValueError("'{}' is not an input name for widget {}"
                                  .format(input, type(widget).__name__))
+        if widget.isBlocking():
```

Comment:
Yes. The (existing) wait is done after the inputs are set and `handleNewSignals` is called. It is somewhat misplaced , but I do not want to change it.